test/contrib/test_module.py

```diff
@@ -311,3 +315,360 @@ def model():
     guide = AutoDelta(model)
     svi = SVI(model, guide, numpyro.optim.Adam(0.01), Trace_ELBO())
     svi.run(random.PRNGKey(100), 10)
+
+
+@pytest.mark.skipif(sys.version_info[:2] == (3, 9), reason="Skipping on Python 3.9")
+def nnx_model_by_shape(x, y):
```

Comment:
you can remove this test now I guess. Not sure why we can still construct the module lazily.

Reply:
Done https://github.com/pyro-ppl/numpyro/pull/1990/commits/e9d6b83cac59c2333c727c14bd0be2a5c8f277d8 !